daemon/web/components/TerminalView/TerminalView.js

```diff
@@ -1,26 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
-const styles = {
```

Comment:
I noted in my other PR that this might be best to keep here, so as to allow programmatic styling via themes, but I suppose this is fine for now